docs/pages/management/security/privilege-elevation.mdx

```diff
@@ -0,0 +1,68 @@
+---
+title: Privilege Elevation
+description: Explains the risks of root-level access to Teleport-protected resources.
+---
+
+As an administrator, you need to make informed decisions about when and how to grant
+privileged access to the resources in your infrastructure, including the Teleport services
+that protect those resources. You also need to manage access to the cluster storage backend
+and determine whether to apply network-level restrictions.
+
+## Labels and label expressions
+
+As you learned in [Add Labels to Resources](../../management/admin/labels.mdx), labels enable you to group, filter, and control access to 
+nodes and services in your infrastructure that you want to protect with Teleport. 
+When combined with allow and deny rules in Teleport roles, labels become the primary mechanism 
+for granting or denying access to critical resources.
+Users who can add or modify labels can—either intentionally or unintentionally—make changes 
+that result in privilege elevation when other users' roles are evaluated. 
+
+To prevent changes to labels from granting elevated privileges, you should:
+
+- Carefully consider how allow and deny rules are defined and interact in every role.
+- Restrict access to the Teleport process and Teleport configuration files on all nodes in the cluster.
+- Use care in defining dynamic labels that execute commands.
+- Restrict access to any command that may be run as part of dynamic labels.
```

Comment:
I would reword this to something like:
` - Consider how dynamic labels are sourced, restrict possible modifications to the command or the data sources used`